dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -315,12 +317,16 @@ public Organization updateOrganizationDescription(@ApiParam(hidden = true) @Para
     @Operation(operationId = "getOrganizationMembers", summary = "Retrieve all members for an organization.", description = "Retrieve all members for an organization. Supports optional authentication.", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
     public Set<OrganizationUser> getOrganizationMembers(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth Optional<User> user,
         @ApiParam(value = "Organization ID.", required = true) @Parameter(description = "Organization ID.", name = "organizationId", in = ParameterIn.PATH, required = true) @PathParam("organizationId") Long id) {
-        Set<OrganizationUser> acceptedUsers = getOrganizationByIdOptionalAuth(user, id).getUsers()
-            .stream()
-            .filter(orgUser -> orgUser.isAccepted())
-            .collect(Collectors.toSet());
+        Organization organization = getOrganizationByIdOptionalAuth(user, id);
+
+        if (user.isPresent() && isUserAdmin(organization, user.get().getId())) {
+            return organization.getUsers(); // Organization admins can view accepted, pending, and rejected users
```

Comment:
According to the [docs](https://docs.dockstore.org/en/stable/advanced-topics/organizations-and-collections.html?highlight=organizations), only admins can update membership.

If that's the case, should maintainers be able to view rejections/pending? I'd lean towards not, but I don't feel too strongly about it, just raising the question.

Reply:
This uses `isUserAdmin` instead of `isUserAdminOrMaintainer` so only organization admins can view rejections/pending.